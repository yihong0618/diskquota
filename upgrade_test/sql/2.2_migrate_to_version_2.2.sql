\! gpconfig -c shared_preload_libraries -v 'diskquota-2.2.so' > /dev/null
\! gpstop -raf > /dev/null

\! gpconfig -s 'shared_preload_libraries'

\c
alter extension diskquota update to '2.2';
\! sleep 5
