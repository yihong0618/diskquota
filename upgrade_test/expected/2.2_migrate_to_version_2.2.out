\! gpconfig -c shared_preload_libraries -v 'diskquota-2.2.so' > /dev/null
\! gpstop -raf > /dev/null
\! gpconfig -s 'shared_preload_libraries'
Values on all segments are consistent
GUC          : shared_preload_libraries
Master  value: diskquota-2.2.so
Segment value: diskquota-2.2.so
\c
alter extension diskquota update to '2.2';
\! sleep 5
